### PR TITLE
Added close function in the onSuccess method to close widget

### PR DESCRIPTION
**Pull request by mfonisoebong**
Added functionality to close the checkout widget after successful payment

### CODE REVIEW

File: src/index.js
Comment (fujahgabriel) [inline src/index.js:1]: adding a close function to onSuccess alters how onSuccess should work. onSucess, the receipt page becomes active for customers to download the receipt. triggering the close will make this impossible.  Devs should have the freedom to handle close the way they want. Another approach is to trigger the close here.

Also, any reason for removing the semicolons?


```javascript
vellaSDK.onSuccess(response => {
  console.log("data", response) // success response with data
  //handle logic
  vellaSDK.close()
})
```

Comment (mfonisoebong) [reply src/index.js:1]: Okay, I understand the functionalities now. As for the semicolons, it was my editor (specifically my prettier extensions configurations) that made that happen. I suggest that a .prettier file is added to the project to ensure that the expected prettier configurations work across all contributors code editors.
Thanks for reviewing the pull request.

Comment (mfonisoebong) [reply src/index.js:1]: `export default class VellaCheckout {
    constructor(key: any, options: any);
    config: {
        _publicKey: any;
        options: any;
    };
    init: () => VellaCheckout;
    onError: (callback: any) => void;
    onSuccess: (callback: any) => void;
    onClose: (callback: any) => void;
}`

I also went thorough the source code and also found out that there's no type for a vellaSDK.close() neither is there an implementation for it

Comment (fujahgabriel) [reply src/index.js:1]: It was a suggestion for you to implement if you wanted to contribute.

Comment (mfonisoebong) [reply src/index.js:1]: Thanks for the opportunity. I'll do my possible best to make sure that I implement the appropriate features where they are needed

**Pull request closed without merging**